en/drive_client/drive_client_for_macos.md

```diff
@@ -0,0 +1,74 @@
+# SeaDrive for macOS and later versions
+
+If you are using macOS 12.1 or later versions, then you can download SeaDrive 3.0.
+
+Please download SeaDrive 2.0 if your macOS is below 12.1. All SeaDrive clients require macOS 10.14 or higher to install and run.
+
+SeaDrive 3.0 use File Provider API to implement SeaDrive File Provider extension, which is redesigned with deeper integration to macOS. We recommend macOS users to upgrade to SeaDrive 3.0 for more native virtual drive experience.
+
+## Install and Access the Virtual Drive
+
+You can download SeaDrive 3.0 from [official Seafile website](https://www.seafile.com/en/download/). After installation and logging into your Seafile account, SeaDrive creates a SeaDrive entry in the Finder. Upon clicking the entry, SeaDrive promptly presents the categorized folder, making it ready for use.
```

Comment:
Upon clicking the entry, SeaDrive extension will be activated. After that you should see category folders such as "My Libraries" in the SeaDrive folder.